packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -592,6 +605,29 @@ - (BOOL)hasCameraRollPermission
   return [_permissionsManager hasGrantedPermissionUsingRequesterClass:[EXImagePickerCameraRollPermissionRequester class]];
 }
 
+
+- (NSString *)mapJSPresetToNativePreset:(NSNumber *)preset API_AVAILABLE(ios(11));
```

Comment:
I'd use `import`/`export` naming convention for such methods. In that case it would be:
```suggestion
- (NSString *)importExportPreset:(NSNumber *)preset API_AVAILABLE(ios(11));
```
But `importExport` sounds weird, though 😅